src/config_parser_utils/merge_configuration_files.py

```diff
@@ -15,59 +19,105 @@
 """
 import configparser
 import logging
+from logging.handlers import SysLogHandler
+import shutil
 import sys
 import os
+import re
+
+CFG_LINE_RGX = r"^(\S+)\s*=\s*(.*)$"
+
+def setup_logger():
+    """Configures and returns a logger that sends logs to syslog."""
+    logger = logging.getLogger('upgrade')
+    logger.setLevel(logging.DEBUG)
```

Comment:
no need to have debug severity, info is enough
```suggestion
    logger.setLevel(logging.INFO)
```

Reply:
It would be best to have it configurable

Reply:
fixed